### PR TITLE
hfstospell 0.5.1

### PR DESCRIPTION
**Pull request by osimola**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
According to discussion at https://github.com/hfst/hfst-ospell/issues/48, the package isn't really supposed to be buildable without disabling libxml++.

### CODE REVIEW

**Comment by chenrui333** [comment]
10:39:55 ==> brew linkage --test libvoikko
10:39:57 ==> FAILED
10:39:57 Broken dependencies:
10:39:57   /usr/local/opt/hfstospell/lib/libhfstospell.10.dylib (hfstospell)

**Comment by chenrui333** [comment]
You might need a revision bump for `libvoikko`

**Comment by osimola** [comment]
Done. I don't see any reason why one would be needed as there are no API changes, but hfstospell maintainer thinks otherwise or just doesn't care.

**Comment by chenrui333** [comment]
Looks like the build still failed, i might be wrong on the revision bump side.

File: Formula/hfstospell.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Hfstospell < Formula
   desc "Helsinki Finite-State Technology ospell"
   homepage "https://hfst.github.io/"
-  url "https://github.com/hfst/hfst-ospell/releases/download/v0.5.0/hfstospell-0.5.0.tar.gz"
-  sha256 "0fd2ad367f8a694c60742deaee9fcf1225e4921dd75549ef0aceca671ddfe1cd"
+  url "https://github.com/hfst/hfst-ospell/releases/download/v0.5.1/hfstospell-0.5.1.tar.gz"
+  sha256 "ccf5f3b06bcdc5636365e753b9f7fad9c11dfe483272061700a905b3d65ac750"
   revision 6
```
Comment (chenrui333) [inline Formula/hfstospell.rb]:
```suggestion
```

Comment (chenrui333) [reply Formula/hfstospell.rb]: That line can be removed.

Comment (osimola) [reply Formula/hfstospell.rb]: Done

**Comment by chenrui333** [comment]
Thanks @osimola!!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**